aten/src/ATen/function_wrapper.py

```diff
@@ -157,6 +169,17 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 #endif
 }
 """)
+C10_TENSOR_METHOD_DEFINITION = CodeTemplate("""\
+inline ${return_type} Tensor::${api_name}(${method_formals}) const {
+#ifdef USE_STATIC_DISPATCH
+    ${static_dispatch_method_body}
+#else
+    static c10::OperatorHandle op = c10::Dispatcher::singleton().findSchema({"aten::${name}", "${overload_name}"}).value();
+    return c10::Dispatcher::singleton().callUnboxed<${formals_types_with_return}>(
+        op, impl::dispatchTypeId(type_set())${method_actuals_with_comma_prefix});
```

Comment:
This gonna need to do the multiple dispatch thing.

Reply:
ouch I think this got screwed up by a merge conflict with the revert of the multi dispatch diff.